modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -63,14 +67,22 @@ DNS resolution prior to making the change.
 +
 Note that Runtime Fabric can use up to the maximum number shown if the cores are available. Transport Layer Security (TLS) termination is computationally expensive, so allocate enough CPU to increase throughput and decrease latency. Refer to See the xref:deploy-resource-allocation.adoc[Resource Allocation and Performance on Anypoint Runtime Fabric] for more information on CPU allocation and throughput, including how the TLS private key type and size affect these numbers.
 .. Specify the minimum memory to allocate for each controller node for the internal load balancer. Note that Runtime Fabric can use up to the maximum amount shown if it is available. 
-. Configure a TLS context.
+. Configure a TLS context for HTTPS.
++
+If you selected *Accept HTTPS requests* in the *Basic Configuration* section, all inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. In this case, when enabling inbound traffic, you can provide up to 10 certificates, but at least one valid TLS certificate is required.
 +
-All inbound traffic entering Anypoint Runtime Fabric is encrypted using TLS. When enabling inbound traffic, you must provide a valid TLS certificate. After inbound traffic is enabled, HTTP requests sent to Runtime Fabric receive a `301` response to redirect to HTTPS on port 443.
+The Protocol option that you selected determines how HTTP and HTTPS requests are handled:
```

Comment:
It feels strange to have the guidance for the Protocols UI split between two sections.

Reply:
Good catch - I've moved this.